### DISCUSSION TITLE
Asset Tag not required

### DISCUSSION THREAD

**Discussion by sup3rt3dy** [snipe/snipe-it#13032] [Q&A]
We are just establishing Asset Management and our asset don't have Asset Tag number, and its also not planed to have them on mobile phones.

Is it possible to disable that asset tag is required?

**Comment by devvonmonahan** [comment]
How about just turn autoincrement on and it will select the asset tag itself?
Then you can just create assets and ignore it

This also helps in the future if you start wanting to refer to assets via their asset tag.

That's what I am doing at the moment.

**Reply by sup3rt3dy** [reply]
was thinking of doing something like that... :D

**Comment by qay21** [comment]
Agree with @CrypNZ, just let it autoincrement and forget about it. We had the same situation as you a few months ago when starting to use SnipeIT : we tend to refer to assets either with their Serial Number when exactness is required, or through vague descriptions like "Frank's laptop" otherwise. So initially we didnt exactly needed asset tags, and simply let the 5-digits value with autoincrement.

But now that it is running, we tend to use asset tags more and more actually : tags are way way faster to right down on a sticky note than S/N. Plus, some assets are very good at hiding their S/N, like Logitech mouse having a false S/N on the box and the real S/N printed inside the battery compartment, under the battery itself.

So yeah, autoincremented tags, and maybe at some point you might catch yourself using tags for quick identification of various stuff
